content/de/es6.md

```diff
@@ -1,5 +1,81 @@
-<!-- Pending translation -->
- <meta http-equiv="refresh" content="0;/en/es6.html">
- <script>window.location.href='/en/es6.html';</script>
+# ES6 und io.js
 
-<!-- de i18n team, feel free to put a redirect notice here -->
+Die io.js-Entwicklung basiert auf der aktuellen Version der [V8](https://code.google.com/p/v8/)-Engine. Mit der Bestrebung, ständig die aktuellste Version der V8-Engine einzusetzen, stellen wir sicher, dass neue Funktionen der [JavaScript ECMA-262 Spezifikation](http://www.ecma-international.org/publications/standards/Ecma-262.htm) sowie Geschwindigkeits- und Stabilitätsverbesserungen schnell für io.js-Entwickler zu Verfügung stehen.
+
+Die Version {{project.current_version}} io.js wird mit der V8-Version {{project.current_v8}} ausgeliefert, welche bereits ES6-Funktionen beinhaltet, die deutlich weiterentwickelter sind, als jene der Version 3.28.73, die in joyent/node@0.12.x enthalten sind.
```

Comment:
"Die Version {{project.current_version}} *von* io.js"

Suggestion:
"die deutlich weiterentwickelter sind, als die der Version 3.28.73, welche in joyent/node@0.12.x enthalten ist."